### PR TITLE
UCM/TEST-EVENT: added special processing of aggregate events

### PR DESCRIPTION
**Pull request by hoopoepg**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9293/ for details (Mellanox internal link).

File: src/ucm/event/event.c [hunk 1]
```diff
@@ -588,6 +588,17 @@ ucs_status_t ucm_test_events(int events)
 {
     int out_events;
 
+    if (events & UCM_EVENT_VM_MAPPED) {
+        events |= UCM_EVENT_MMAP | UCM_EVENT_MREMAP |
```
Comment (brminich) [inline src/ucm/event/event.c]: Maybe create a macro aggregating these events? 
At least ucm_event_install uses the same sets

Comment (hoopoepg) [reply src/ucm/event/event.c]: good idea... done

File: src/ucm/event/event.c [hunk 2]
```diff
@@ -26,6 +26,15 @@
 #include <string.h>
 #include <errno.h>
 
+
+#define UCM_NATIVE_EVENT_VM_MAPPED (UCM_EVENT_MMAP  | UCM_EVENT_MREMAP | \
+                                    UCM_EVENT_SHMAT | UCM_EVENT_SBRK)
+
+#define UCM_NATIVE_EVENT_VM_UNMAPPED (UCM_EVENT_MMAP   | UCM_EVENT_MUNMAP | \
```
Comment (brminich) [inline src/ucm/event/event.c]: minor: you could use `UCM_NATIVE_EVENT_VM_MAPPED ` here, because UNMAPPED set is an extension of MAPPED

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9296/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9297/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9302/ for details (Mellanox internal link).

**Comment by hoopoepg** [comment]
failed due to git issue

**Comment by hoopoepg** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9304/ for details (Mellanox internal link).

**Comment by hoopoepg** [comment]
@yosefe ok to merge?

**Pull request merged**